tests/memex/parse_document_claims_test.py

```diff
@@ -261,6 +265,36 @@ def test_document_metas_from_data_with_multiple_metadata_claims(self):
                 'claimant': claimant,
                 } in document_metas
 
+    def test_document_metas_from_data_ignores_null_titles(self):
+        """It should ignore null document titles."""
+        for title in (None, [None, None]):
+            document_data = {'title': title}
+
+            document_metas = parse_document_claims.document_metas_from_data(
+                document_data, 'http://example/claimant')
+
+            assert document_metas == []
+
+    def test_document_metas_from_data_ignores_empty_string_titles(self):
+        """It should ignore empty document titles."""
+        for title in ('', ['', '']):
+            document_data = {'title': title}
+
+            document_metas = parse_document_claims.document_metas_from_data(
+                document_data, 'http://example/claimant')
+
+            assert document_metas == []
+
+    def test_document_metas_from_data_ignores_whitespace_only_titles(self):
+        """It should ignore whitespace-only document titles."""
+        for title in (' ', [' ', ' '], '\n\n  \n'):
+            document_data = {'title': title}
+
+            document_metas = parse_document_claims.document_metas_from_data(
+                document_data, 'http://example/claimant')
+
+            assert document_metas == []
```

Comment:
Considering my comment above, and if this behaviour should only apply to titles, then I think it would be good to have a test which shows that we don't ignore whitespace-only values for other types than `title`.